### ISSUE TITLE
[css-map] pagelinks broken & sidebar config no longer functions

### ISSUE THREAD

**Issue by ohitstom** [spicetify/spicetify-cli#2977]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
Spotify for Windows (64 bit)
1.2.36.955.gd8cc9372
Spicetify v2.36.6
xpui_2024-04-19_1713551583143_d8cc937
cef_122.1.13+gde5b724+chromium-122.0.6261.130
Runtime: Chrome
```

### 📝 Description

see issue in screenshot note this is the same with or without a theme

### 📸 Screenshots

![image](https://github.com/spicetify/spicetify-cli/assets/22730962/5ef5ff4c-d3b0-4b25-8a11-0413157b5509)
![image](https://github.com/spicetify/spicetify-cli/assets/22730962/cfc4737f-d664-4109-8864-0e0d5c34bbf3)

**Comment by rxri** [comment]
I forgot to map these.

**Comment by ohitstom** [comment]
please look into these again, i can assure you these were mapped one spotify version down from latest - alot of extensions rely on them and themes this is how i know
![image](https://github.com/spicetify/spicetify-cli/assets/22730962/2903eb71-cf3c-4d23-8ba2-4e36ec084dba)

**Comment by rxri** [comment]
is it really that hard to do it by yourself? omfg

**Comment by ohitstom** [comment]
> is it really that hard to do it by yourself? omfg

since you are mapping anyways wouldn't this be easier  to do at the same time 🤔 if you are going to put maintainer in your bio maybe you should actually maintain - its not like im requesting you to implement a new feature

and its not like im the only dev that relies on these classes - so maybe dont close an issue that isnt resolved if devs need to pitch in otherwise they will be unaware.

**Comment by rxri** [comment]
I asked you to do the pull request because I noticed your comment after closing the issue, but you ignored me it seems and decided to put everything on me. I'm a maintainer of it sure, but this is open source project and I don't need to do every thing in it - because everyone can contribute it. At this point, I will look into it.

**Comment by rxri** [comment]
Didn't think of it and you're also blocked on my end so I do not get notifications when you're commenting on issues and such I didn't bother looking into the conversation on it.

As of the unmapped class in `topSentinel`, it's empty for me so I don't even have this class. If you're willing to provide me the class that was used in versions before, I will be more than happy to map it.

**Comment by ohitstom** [comment]
> As of the unmapped class in `topSentinel`, it's empty for me so I don't even have this class. If you're willing to provide me the class that was used in versions before, I will be more than happy to map it.

that is very strange, i will open a pr myself later since that will be easier.

**Comment by rxri** [comment]
![image](https://github.com/spicetify/spicetify-cli/assets/9348108/3c5c6936-3d76-4444-a569-e1152dfbfcf1)
I don't even know about what elements we're talking because listitems have multiple classes, not only one.

**Comment by rxri** [comment]
Ah, I see now. Will map these

**Comment by ohitstom** [comment]
> Ah, I see now. Will map these

sorry i should have been more informative, glad you see the issue too

**Comment by ohitstom** [comment]
> Ah, I see now. Will map these

sorry to bother again but this weird text issue still seems to be present, should i open a new issue?
![image](https://github.com/spicetify/spicetify-cli/assets/22730962/7ffa52c6-3422-4e14-88ea-6beff85a2061)
![image](https://github.com/spicetify/spicetify-cli/assets/22730962/30535422-ffd4-4a80-8fbd-9827e9b02309)
